Add how to find the management API Key (#686)
Amplitude-Developer-Docs/amplitude-dev-center: docs/experiment/apis/management-api.md

    This API is in Beta. If you would like to try out an early version of the API please reach out to us.

## Management API Key

> You can manage your management API keys by going to `https://experiment.amplitude.com/<org-url>/management-api` or by clicking on "Management API" in the bottom left of the sidebar. Here you can create keys, revoke keys, and delete keys. You can read more about the Management API Key [here](../../../guides/amplitude-keys-guide/?h=keys#management-api-key).

Reviewer: ```suggestion
You can create and revoke management API keys by clicking on "Management API" in the bottom left of the sidebar. Read more about the Management API Key in our [comprehensive guide to all amplitude keys](../../../guides/amplitude-keys-guide/?h=keys#management-api-key).
```
